CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ The public API of this library consists of the functions declared in file
 
 ## [Unreleased]
 
+### Added
+- closed-form implementation of `numHexagons`
+
+### Fixed
+- `maxH3ToChildrenSize` to output `int64_t` instead of `int` (breaking change)
+
```

Comment:
Maybe put breaking changes in their own section. This will become the changelog for 4.0.0.